client/src/component/sos/audit/AuditPage.tsx

```diff
@@ -22,13 +22,18 @@ const Breadcrumb = () => (
     </ul>
 );
 
+function round(val: number, digits: number) {
+    const factor = Math.pow(10, digits);
+    return Math.round(val * factor) / factor;
+}
+
 const ReadonlyRiskLimit = ({ riskLimit }: any) => {
-    const riskLimitPercent = Math.round(riskLimit * 100);
+    const riskLimitPercent = round(riskLimit * 100, 2);
```

Comment:
Can you explain to me the decision to use a `digits` of 2?

Reply:
Yes. We use `digits` to compute a shift factor. The shift factor is used to shift the decimal point left or right, via division or multiplication. We truncate all sub-decimal values with `Math.round`, then shift back. So, for `d == 2`, we `factor == 100`, and for, say, `riskLimit == 1.234`, we obtain: `Math.round(1.234 * 100) / 100` -> `Math.round(123.4) / 100` -> `123 / 100` -> `1.23`.

Reply:
Open to revision. This allows two digits past the decimal point, when regarding the risk limit as a percentage. So this allows `12.34 %`, or `0.1234`. We could narrow it if desired. I just chose the maximum precision that seemed remotely reasonable, expecting to discuss it in review.

Reply:
That's what I thought.  I certainly think that two digit precision is sufficient as a default.